data/sql_updates/create_candidate_history_view.sql

```diff
@@ -0,0 +1,44 @@
+drop materialized view if exists ofec_candidate_history_mv;
+create materialized view ofec_candidate_history_mv as
+select
+    dcp_recent.candproperties_sk as properties_key,
```

Comment:
Compared to how we usually use "recent," I feel like it is a bit misleading in this context, since it will go back through time. Can we alias the table to something like "dcp_by_period"?